composer.json

```diff
@@ -3,15 +3,15 @@
     "description": "Helps to discover and install tools",
     "type": "project",
     "require": {
-        "php": "~8.2.0 || ~8.3.0 || ~8.4.0",
-        "symfony/console": "^6.4.18 || ^7.2",
+        "php": "~8.2.0 || ~8.3.0 || ~8.4.0 || ~8.5.0",
+        "symfony/console": "^6.4.18 || ^7.2 || ^8.0",
         "psr/container": "^2.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^11.5.9 || ^12.0",
         "zalas/phpunit-globals": "^4.0",
-        "infection/infection": "^0.29",
-        "deptrac/deptrac": "^3.0"
+        "infection/infection": "^0.31",
+        "deptrac/deptrac": "^4.0"
```

Comment:
Are these updates necessary? If so, maybe it would be easier to submit these as separate PRs, as some of these break the build?